pom.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>95</version>
+    </parent>
+
+    <groupId>io.prestosql.tempto</groupId>
+    <artifactId>tempto-root</artifactId>
+    <version>172-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <description>Tempto - test framework</description>
+    <url>https://github.com/airlift/drift</url>
```

Comment:
please update

Reply:
Thanks, missed that. You see where I copied this from :)